modules/vulnerability-management-review-component-state.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_module-type: PROCEDURE
```

Comment:
@gaurav-nelson since this is the last minute pull from this release its simply a matter of deleting this module and the context surrounding it specifically.